apps/akasha/src/components/app-routes.tsx

```diff
@@ -18,14 +19,10 @@ const { Box } = DS;
 
 const AppRoutes: React.FC<RootComponentProps> = props => {
   const { logger } = props;
-
   const [, errorActions] = useErrors({ logger });
```

Comment:
I've noticed this pattern used a lot, seems to only be used to log errors, but it does cause a needless rerender of the component. Not a big deal, but IMO logging should maybe be separated into its own custom hook? 🤔